massa-execution-worker/src/execution.rs

```diff
@@ -1527,6 +1527,25 @@ impl ExecutionState {
         ops
     }
 
+    /// Check if a denunciation has been executed given a `DenunciationIndex`
+    pub fn is_denunciation_unexecuted(&self, denunciation_index: &DenunciationIndex) -> bool {
+        // check active history
+        let history = self.active_history.read();
+
+        if matches!(
+            history.fetch_processed_de(denunciation_index),
+            HistorySearchResult::Present(())
+        ) {
+            return true;
```

Comment:
I think "unexecuted" should be changed to "executed" everywhere because it causes confusion.
Here for example it caused a `return true` but the denunc WAS executed

Reply:
Yes, always avoid double negation :)